docs/notebooks/index.rst

```diff
@@ -6,7 +6,6 @@ Examples
    :maxdepth: 1
 
    example
-   subset
```

Comment:
I wouldn't remove the subset example yet until clisops documentation is online accessible and shows how this is performed.

Reply:
the clisops docs has a copy of the subset example notebook:
https://clisops.readthedocs.io/en/latest/notebooks/subset.html

Should it be linked to the xclim docs?

Reply:
That's good to see. Seeing as we want users to start using clisops in the near future, it might be good to place a note somewhere that links to the clisops docs. The one thing that we might want to do for the subset example is to run it and leave the output intact so people can see the figures. I can look into porting that to theclisops docs if you want.